GTG/gtk/ui/preferences.ui

```diff
@@ -0,0 +1,497 @@
+                                <property name="can_focus">False</property>
+                                <property name="xalign">0.0</property>
+                                <property name="label" translatable="yes">Set day-start time</property>
+                                <attributes>
+                                  <attribute name="weight" value="bold"/>
+                                </attributes>
+                              </object>
+                            </child>
+                            <child>
+                              <object class="GtkLabel" id="time_label">
+                                <property name="label">Set time which will be used as a start of the new day in GTG</property>
+                                <property name="visible">true</property>
+                                <property name="xalign">0.0</property>
+                                <style>
+                                  <class name="dim-label"/>
+                                </style>
+                                <attributes>
+                                  <attribute name="scale" value="0.833333"/>
+                                </attributes>
+                              </object>
+                            </child>
+                          </object>
+                          <packing>
+                            <property name="position">3</property>
+                          </packing>
+                        </child>
+                      </object>
+                      <packing>
+                        <property name="expand">False</property>
+                        <property name="fill">True</property>
+                      </packing>
+                    </child>
+                    <child>
+                      <object class="GtkLabel" id="editor_label">
+                        <property name="visible">true</property>
+                        <property name="label" translatable="yes">Task Editor</property>
+                        <property name="xalign">0.0</property>
+                        <property name="margin-start">10</property>
+                        <property name="margin-top">6</property>
+                        <style>
+                          <class name="dim-label"/>
+                        </style>
+                        <attributes>
+                          <attribute name="weight" value="bold"/>
+                        </attributes>
+                      </object>
+                    </child>
+                    <child>
+                      <object class="GtkBox" id="font_box">
+                        <property name="margin-start">40</property>
+                        <property name="margin-end">20</property>
+                        <property name="margin-top">10</property>
+                        <child>
+                          <object class="GtkFontButton" id="font_button">
```

Comment:
Should we make the font button make it 100% wide as in GNOME builder?

![sample](https://people.gnome.org/~chergert/screenshots/gnome-builder-3.16.3/3.png)